doc/src/arch/reference.rst

```diff
@@ -1122,6 +1122,8 @@ The following tags are common to all ``<tile>`` tags:
 
         .. note:: If the ``<pinlocations>`` tag is missing, a ``spread`` pattern is assumed.
 
+        .. warning:: If a pin is defined on multiple locations, only first location will be considered in routing.
```

Comment:
Should this read "If a pin with equivalence is defined ..."?

Should VPR detect and generate an error / warning in case the arch defines this case?

Reply:
In the latest fix-up, we can properly handle the multiple locations for pins. I will remove this warning.